AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image.js

```diff
@@ -8,14 +8,20 @@ define([
     'use strict';
 
     return Column.extend({
+        defaults: {
+            modules: {
+                previewComponent: '${ $.parentName }.preview'
```

Comment:
Image component dependency on preview component makes it impossible to create a masonry grid without preview. Could we reverse the dependency?

Reply:
I'm not sure that we can listen to events from the another component but I'll double-check and make the changes if it's possible